db/README.md

```diff
@@ -351,10 +344,12 @@ HAVING (count(zipped_moab_versions.id) < current_version)
 
 _**note**: might take a few minutes to run on prod_
 
+TODO: Figure out how to replace this.
+
 ```ruby
 [53] PreservedObject.joins(preservation_policy: [:zip_endpoints]).left_outer_joins(:zipped_moab_versions).group(:druid, :current_version, :endpoint_name).having('count(zipped_moab_versions.id) = 0').pluck(:druid, :current_version, :endpoint_name, 'count(zipped_moab_versions.id)')
 ```
```

Comment:
@jmartin-sul I could use a bit of help getting the replaced query here right.

Reply:
FWIW, I ran the current query and these two queries in stage and got different results. Original query has 4,683 results. The `ZipEndpoint` version came back with 357 results. The `PreservedObject` version came back with 1,561 results. Tested these with `main` in stage. Not sure how to handle this delta in this PR which I just rebased. Maybe leave as TODO for follow-on work?